etl/steps/data/garden/artificial_intelligence/2023-07-25/us_cpi.py

```diff
@@ -0,0 +1,63 @@
+import xml.etree.ElementTree as ET
+
+import pandas as pd
+import requests
+from structlog import get_logger
+
+log = get_logger()
+
+# change number of years to go back if neccessary (used up to 2000 in 2023)
+
+YEARS = 23
+
+
+def import_US_cpi_API(years_back=YEARS):
```

Comment:
This is very cool that you are using this data! I think it might be better to get it from the WB dataset in the etl like you did for [the UNWTO dataset](https://github.com/owid/etl/blob/57113df3bd20f04de04aafd5aa37b8d08a60ed2d/etl/steps/data/garden/tourism/2023-05-05/unwto.py). There's just the slight advantage that we'll definitely know what version we are using and can be more sure it won't break unexpectedly.

Reply:
The reason why I am doing this here is because our WB dataset only goes up to 2021 whereas we can get the data for 2022 too via the API . So this is just a workaround I've used until our WB dataset is updated for my AI datasets in general :) Though it looks like the data in this one only goes up until 2021 anyway so I can also just use the WB dataset for now.

Reply:
Ahhh that makes sense!